index.html

```diff
@@ -295,6 +295,35 @@ <h3 class="lang">Skills</h3>
       <img class="saperator" src="images/Separator Bottom.png" alt="">
   </section>
 
+  <section class="contact_section">
+
+    <h2 class="contact-head">Contact me</h2>
+    <p class="contact_text">If you have an application you are interested in developing, a feature that you need built
+      or a project that
+      needs coding. I’d love to help with it </p>
+
+    <form action="https://formspree.io/f/xleknroo" method="POST">
+      <form id="fs-frm" class="contact_form" name="simple-contact-form" action="https://formspree.io/f/{form_id}"
+        method="post">
+
+        <label for="full-name">
+          <input class="input_name" type="text" name="name" id="full-name" placeholder="Maurice"
+            accept-charset="utf-30">
+        </label>
+        <label>
+          <input class="input_mail" type="email" name="email" placeholder="Olabisi2k11@gmail.com">
+        </label>
+        <label>
+          <textarea class="message_box" name="message" placeholder="write your message here"></textarea>
+        </label>
```

Comment:
Great job so far 👍 . Kindly make sure all the input field has the `required` attribute to activate the validation on submit. 😊